src/material/paginator/paginator.spec.ts

```diff
@@ -23,52 +23,64 @@ describe('MatPaginator', () => {
   }
 
   describe('with the default internationalization provider', () => {
-    it('should show the right range text', () => {
-      const fixture = createComponent(MatPaginatorApp);
-      const component = fixture.componentInstance;
-      const rangeElement = fixture.nativeElement.querySelector('.mat-paginator-range-label');
-
-      // View second page of list of 100, each page contains 10 items.
-      component.length = 100;
-      component.pageSize = 10;
-      component.pageIndex = 1;
-      fixture.detectChanges();
-      expect(rangeElement.innerText.trim()).toBe('11 – 20 of 100');
-
-      // View third page of list of 200, each page contains 20 items.
-      component.length = 200;
-      component.pageSize = 20;
-      component.pageIndex = 2;
-      fixture.detectChanges();
-      expect(rangeElement.innerText.trim()).toBe('41 – 60 of 200');
-
-      // View first page of list of 0, each page contains 5 items.
-      component.length = 0;
-      component.pageSize = 5;
-      component.pageIndex = 2;
-      fixture.detectChanges();
-      expect(rangeElement.innerText.trim()).toBe('0 of 0');
-
-      // View third page of list of 12, each page contains 5 items.
-      component.length = 12;
-      component.pageSize = 5;
-      component.pageIndex = 2;
-      fixture.detectChanges();
-      expect(rangeElement.innerText.trim()).toBe('11 – 12 of 12');
-
-      // View third page of list of 10, each page contains 5 items.
-      component.length = 10;
-      component.pageSize = 5;
-      component.pageIndex = 2;
-      fixture.detectChanges();
-      expect(rangeElement.innerText.trim()).toBe('11 – 15 of 10');
-
-      // View third page of list of -5, each page contains 5 items.
-      component.length = -5;
-      component.pageSize = 5;
-      component.pageIndex = 2;
-      fixture.detectChanges();
-      expect(rangeElement.innerText.trim()).toBe('11 – 15 of 0');
+    describe('showing the right range text', () => {
+      let fixture: ComponentFixture<MatPaginatorApp>;
+      let component: MatPaginatorApp;
+      let rangeElement: HTMLElement;
+
+      beforeEach(() => {
+        fixture = createComponent(MatPaginatorApp);
```

Comment:
I've been avoiding having a single `beforeEach` like this, because it makes it difficult to add another test that doesn't check the `MatPaginatorApp`. It's a little repetitive, but we should create the fixture in each test instead.